charts/argo-events/templates/argo-events-webhook/deployment.yaml

```
@@ -11,7 +11,7 @@ spec:
   selector:
     matchLabels:
       {{- include "argo-events.selectorLabels" (dict "context" . "name" .Values.webhook.name) | nindent 6 }}
-  revisionHistoryLimit: 5
+  revisionHistoryLimit: {{ .Values.controller.revisionHistoryLimit }}
```

Maybe wrong `revisionHistoryLimit` ?
```suggestion
  revisionHistoryLimit: {{ .Values.webhook.revisionHistoryLimit }}
```